lib/files.go

```diff
@@ -0,0 +1,194 @@
+package lib
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// AppendIfMissing appends the content string to the given file
+func AppendIfMissing(filePath string, content string) error {
+	fileToModify, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	Check(err)
+
+	fileContents, err := ioutil.ReadFile(fileToModify.Name())
+	Check(err)
+
+	if !strings.Contains(string(fileContents), content) {
+		_, err = fileToModify.Write([]byte(content))
+	}
+
+	fileToModify.Close()
+
+	return err
+}
+
+// RenameDir renames a directory while handling cross-link issues
+// credit https://github.com/rawlingsj/jx/blob/master/pkg/util/files.go
+func RenameDir(src string, dst string, force bool) (err error) {
+	err = CopyDir(src, dst, force)
+	if err != nil {
+		return fmt.Errorf("failed to copy source dir %s to %s: %s", src, dst, err)
```

Comment:
💡 You might want to check out the `errors` package that we use in a few other places, it provides a lot of nice helpers like, error.Wrapf:

```suggestion
		return errors.Wrapf(err, "failed to copy source dir %s to %s", src, dst)
```

Reply:
This is the one we use in Automate: https://github.com/pkg/errors